lunavl/sdk/estimators/face_estimators/eyes.py

```diff
@@ -213,6 +220,46 @@ def estimate(
             raise LunaSDKException(LunaVLError.fromSDKError(error))
         return EyesEstimation(eyesEstimation)
 
+    #  pylint: disable=W0221
+    @CoreExceptionWrap(LunaVLError.EstimationEyesGazeError)
+    def estimateBatch(
+        self,
+        transformedLandmarksList: List[Union[Landmarks5, Landmarks68]],
+        warps: List[Union[FaceWarp, FaceWarpedImage]],
+    ) -> List[EyesEstimation]:
+        """
+        Batch estimate mouth state on warps.
+
+        Args:
+            warps: warped image list
+            transformedLandmarksList: transformed landmarks list
+
+        Returns:
+            list of estimated states
+        Raises:
+            LunaSDKException: if estimation failed
+            ValueError: if warps count not equals landmarks count
+        """
+        if len(warps) != len(transformedLandmarksList):
+            raise ValueError("Count of warps not equals count of landmarks")
+        cropper = EyeCropper()
+        eyeRectList = []
+        for idx, landmarks in enumerate(transformedLandmarksList):
+            if isinstance(landmarks, Landmarks5):
+                eyeRectList.append(cropper.cropByLandmarks5(warps[idx].warpedImage.coreImage, landmarks.coreEstimation))
+            else:
+                eyeRectList.append(
+                    cropper.cropByLandmarks68(warps[idx].warpedImage.coreImage, landmarks.coreEstimation)
+                )
```

Comment:
is it work for mixed data (with Landmarks68 and Landmarks5)?

Reply:
yes, `test_estimate_batch_mixed_landmarks` added